models/CoronSegmentator/docs/README.md

```diff
@@ -0,0 +1,63 @@
+# Coronsegmentator
+
+### **Authors**
+
+Y. Ke, MC. Chen, TY. Lin, YC. Chan Foxconn Digital Health AI Team
+
+### **Tags**
+
+Segmentation, CT, Heart, Coronary Artery, USD, MONAI, nnU-Net, 3D Reconstruction
+
+## **Model Description**
+
+Coronsegmentator is an automated pipeline that performs dual-task segmentation on cardiac CT images, focusing on both whole-heart and coronary artery structures. It integrates MONAI’s Auto3DSeg for general cardiac segmentation and a custom nnU-Net model for detailed coronary artery segmentation. The pipeline further converts segmentation results (in STL format) into USD files for downstream 3D visualization and digital twin simulation using NVIDIA Omniverse.
+
+The entire workflow is designed to enable precise, scalable, and automated generation of personalized coronary artery digital twins, supporting clinical planning, AI-based diagnosis, and medical visualization in research or pre-operative planning workflows.
+
+## **Data**
+
+The Coronsegmentator model was trained on ImageCAS, a dataset consisting of annotated images for training/validation respectively.
+
+The input data consists of anonymized 3D CT scans in .nii.gz format. This pipeline was designed to be compatible with preoperative cardiac CT data, including varying scanner vendors (e.g., Siemens, Philips).
+
+Each image is processed by:
+
+1. Segmenting the cardiac chambers using MONAI Auto3DSeg
+
+2. Segmenting coronary arteries using a pretrained nnU-Net model
+
+3. Saving segmentation as STL files and converting them into USD
+
+#### **Preprocessing**
+
+Input: .nii.gz NIfTI format (CT scan)
+
+Resolution normalization handled internally
+
+No manual annotation required for inference
+
+#### **Inference**
+
+```bash
+python -m monai.bundle run --config_file "configs/inference.json"
+```
+
+The ImageCAS dataset is publicly available at:
+https://github.com/XiaoweiXu/ImageCAS-A-Large-Scale-Dataset-and-Benchmark-for-Coronary-Artery-Segmentation-based-on-CT.git''
```

Comment:
Would be better to mention the dataset link in the "Data" section, what do you think?